src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -139,6 +174,26 @@ function _parseTooltipAttributes(element) {
   _parseAttributeWithoutOverride(element, 'content', true, '_content');
 }
 
+/*
+ * Warning for deprecated slot names
+ */
+
+function _warnDeprecatedSlotName(element, deprecatedName, correctName) {
```

Comment:
Similar to above, can we make this take an object of deprecated/correct names instead?